aviatrix/data_source_aviatrix_vpc.go

```diff
@@ -163,6 +163,22 @@ func dataSourceAviatrixVpc() *schema.Resource {
 				Computed:    true,
 				Description: "Azure vnet resource ID.",
 			},
+			"availability_domains": {
+				Type:        schema.TypeSet,
```

Comment:
Do you think we should use TypeList instead? That way users can use indexes to access the values
```hcl
availability_domain = data.aviatrix_vpc.A.availability_domains[0]
```